### PR TITLE
Add users to stories.

### LINKED ISSUE #20: Change stories to require users
Every story should have a user that submitted it.

Stories should not be able to be created if the user doesn't exist or isn't specified.

### PR DESCRIPTION
**Pull request by crertel**
This creates a `belongs_to` relation from stories to users.

This also adds tests requiring that the user authoring a story exist.

This closes #20 .

Please review @RookieOne !

### CODE REVIEW

File: priv/repo/migrations/20160601064540_add_users_to_stories.exs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+defmodule PhoenixLobsters.Repo.Migrations.AddUsersToStories do
+  use Ecto.Migration
+
+  def change do
+  	alter table( :stories ) do	  	
+	  	add :user_id, references(:users)
```
Comment (RookieOne) [inline priv/repo/migrations/20160601064540_add_users_to_stories.exs]: 2 spaces instead of 4 por favor. :)

**Comment by RookieOne** [comment]
Besides the 4 spaces instead of 2 for that migration, it looks super awesome. Good job!

**Comment by crertel** [comment]
Yay, thanks!

Can I go ahead and merge that once I've got the 2/4 thing sorted?

**Comment by RookieOne** [comment]
Yup! that is fine.

**Pull request merged**